packages/core/src/sagas/connectSaga.js

```diff
@@ -36,6 +36,7 @@ function* observeAndPutConnectionStatusUpdate(directLine) {
   }
 }
 
+// TODO: [P2] We should move this check and rectification to DirectLineJS.
```

Comment:
file issue :)

Reply:
Done. https://github.com/microsoft/BotFramework-WebChat/issues/2665

This is meta-issue. I will try to list more responsibilities that should be done in DLJS in this bug. We should start clearing responsibilities.